content/source/docs/enterprise/api/index.html.md

```diff
@@ -28,6 +28,47 @@ There are three kinds of token available:
 - [Team tokens](../users-teams-organizations/service-accounts.html#team-service-accounts) — each team has an associated service account, which can have one API token at a time. This is intended for performing plans and applies via a CI/CD pipeline.
 - [Organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts) — each organization also has a service account, which can have one API token at a time. This is intended for automating the management of teams, team membership, and workspaces. The organization token cannot perform plans and applies.
 
+## Feature Entitlements
+
+TFE is available at multiple pricing tiers (including free), which offer different subsets of TFE's features.
+
+Each organization has a set of _entitlements_ that corresponds to its pricing tier. These entitlements determine which TFE features the organization can use.
+
+If an organization doesn't have the necessary entitlement to use a given feature:
+
+- TFE rejects API requests to any endpoints devoted to that feature, returning a (SPECIFIC ERROR CODE GOES HERE) error.
+- (Anything else? Do some endpoints change their response, e.g. do workspace objects have some properties missing if the org can't perform runs?)
```

Comment:
Nope! JSON response objects remain the same for now (I don't think we will change that, but I could be wrong).